preferences/components/diskbus-scsi/diskbus-scsi.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: diskbus-scsi
```

Comment:
`sata` or `scsi`?

Reply:
In my experience scsi. But maybe saat is good enough these days. IMHO it does not make much of a difference.

Maybe check with the RH virt folks?

Reply:
I will keep `scsi` for now. I'm open to adjust it if required.